src/components/ToolMenu/FeatureInfo/index.spec.tsx

```diff
@@ -15,79 +27,168 @@ jest.mock('../../DisplayField', () => (props: any) => (
     expect(FeatureInfo).not.toBeUndefined();
   });
 
-  describe('<FeatureInfoForm />', () => {
-    it('renders form items based on formConfig', () => {
-      const feature = new OlFeature({
-        foo: 'bar',
-        baz: 42
-      });
-      const formConfig = [
-        {
-          propertyName: 'foo',
-          displayName: 'Foo Label',
-          fieldProps: {}
-        },
-        {
-          propertyName: 'baz',
-          fieldProps: {}
-        }
-      ];
-
-      render(
-        <FeatureInfoForm
-          feature={feature}
-          formConfig={formConfig}
-        />
-      );
-
-      expect(screen.getAllByText('Foo Label')[0]).toBeInTheDocument();
-      expect(screen.getByTitle('baz')).toBeInTheDocument();
-      expect(screen.getAllByTestId('display-field')).toHaveLength(2);
-    });
+  it('renders nothing when map is not available', () => {
+    const { container } = render(
+      <Provider store={store}>
+        <MapContext.Provider value={null}>
+          <FeatureInfo />
+        </MapContext.Provider>
+      </Provider>
+    );
 
-    it('calls resetFields and setFieldsValue on feature change', () => {
-      const feature1 = new OlFeature({ foo: 'bar' });
-      const feature2 = new OlFeature({ foo: 'baz' });
-      const formConfig = [{
-        propertyName: 'foo',
-        fieldProps: {}
-      }];
-
-      const { rerender } = render(
-        <FeatureInfoForm
-          feature={feature1}
-          formConfig={formConfig}
-        />
-      );
-      rerender(
-        <FeatureInfoForm
-          feature={feature2}
-          formConfig={formConfig}
-        />
-      );
-
-      expect(screen.getAllByText('foo')[0]).toBeInTheDocument();
+    expect(container.firstChild).toBeNull();
+  });
+
+  it('renders nothing when featureInfo is disabled', () => {
+    const disabledStore = configureStore({
+      reducer: {
+        featureInfo: () => ({ enabled: false }),
+        selectedFeatures: () => ({})
+      }
     });
 
-    it('renders nothing if formConfig is undefined', () => {
-      const feature = new OlFeature({ foo: 'bar' });
-      render(<FeatureInfoForm feature={feature} />);
-      expect(screen.queryByTestId('display-field')).toBeNull();
+    const { container } = render(
+      <Provider store={disabledStore}>
+        <MapContext.Provider value={map}>
+          <FeatureInfo />
+        </MapContext.Provider>
+      </Provider>
+    );
+
+    expect(container.firstChild).toBeNull();
+  });
+
+  it('renders empty form when no formConfig is set', () => {
+    const feature = new OlFeature({
+      foo: 'bar',
+      baz: 42
     });
 
-    it('passes additional props to Form', () => {
-      const feature = new OlFeature({ foo: 'bar' });
-      const { container } = render(
-        <FeatureInfoForm
-          feature={feature}
-          data-testid="my-form"
-        />
-      );
-      expect(container.querySelector('.feature-info-form')).toBeInTheDocument();
+    render(
```

Comment:
I think these tests and the others test behaviou of the `<FeatureInfoForm />` component and not of the `<FeatureInfo />` one, correct? I might be overlooking something.